SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1913,6 +1913,10 @@ void CSolver::InitiateComms(CGeometry *geometry,
       COUNT_PER_POINT  = nVar*nDim*2;
       MPI_TYPE         = COMM_TYPE_DOUBLE;
       break;
+    case PRIMITIVE:
+      COUNT_PER_POINT  = nPrimVar;
+      MPI_TYPE         = COMM_TYPE_DOUBLE;
+      break;  
     case PRIMITIVE_GRADIENT:
```

Comment:
Do you really need the new MPI comm? Usually we compute the primitives for all points, including the halos, and so there is not need to communicate them, they will automagically be the same in different MPI processes.

Reply:
no, this should have been deleted before. well noted